core/src/main/java/software/amazon/awssdk/core/util/json/JacksonUtils.java

```diff
@@ -24,8 +24,10 @@
 import com.fasterxml.jackson.databind.ObjectWriter;
 import java.io.IOException;
 import java.io.Writer;
+import software.amazon.awssdk.annotations.SdkProtectedApi;
 import software.amazon.awssdk.core.exception.SdkClientException;
 
+@SdkProtectedApi
```

Comment:
Could be another useful public feature

Reply:
Before GA, sure. It's not ready to be public right now, though.